lib/__tests__/extends.test.mjs

```diff
@@ -1,20 +1,19 @@
-'use strict';
+import { fileURLToPath } from 'node:url';
 
-const path = require('path');
+import safeChdir from '../testUtils/safeChdir.js';
+import standalone from '../standalone.js';
 
-const safeChdir = require('../testUtils/safeChdir');
-const configExtendingAnotherExtend = require('./fixtures/config-extending-another-extend.json');
-const configExtendingOne = require('./fixtures/config-extending-one');
-const configExtendingThreeWithOverride = require('./fixtures/config-extending-three-with-override');
-const standalone = require('../standalone');
+import configExtendingAnotherExtend from './fixtures/config-extending-another-extend.json';
```

Comment:
ESM import of `*.json` [looks different](https://nodejs.org/api/esm.html#json-modules).

Shall we use [standard ways](https://www.stefanjudis.com/snippets/how-to-import-json-files-in-es-modules-node-js/) of importing JSON? Or maybe convert JSON to JS?

Reply:
Good point, but this is intentional to minimize diffs. 😓 

I'd like to use Import Assertions (the most straightforward), but unfortunately experimental.

Jest supports Import Assertions like this:

```js
// The test passes ✅ 
import configExtendingAnotherExtend from './fixtures/config-extending-another-extend.json' assert { type: 'json' };
```

However, ESLint complains:

```
/Users/masafumi.koba/git/stylelint/stylelint/lib/__tests__/extends.test.mjs
  6:92  error  Parsing error: Unexpected token assert
```

So, I'd like to leave as-is and rewrite it later. Any thoughts?

Reply:
I reconsidered and introduced a new test utility to keep the code small (caa84e07).